ddtrace/contrib/django/patch.py

```diff
@@ -948,17 +961,18 @@ def _(m):
 
 
 def wrap_wsgi_environ(wrapped, _instance, args, kwargs):
-    if _is_iast_enabled():
+    try:
```

Comment:
Is there a particular reason we're replacing the `_is_iast_enabled()` check with a try/catch?

Reply:
rollback-ed, It was a experiment, TY!